src/common/btree.c

```diff
@@ -48,6 +48,8 @@ enum btr_probe_rc {
 	PROBE_RC_NONE,
 	PROBE_RC_OK,
 	PROBE_RC_ERR,
+	PROBE_RC_INPROGRESS,
```

Comment:
AGAIN is a better fit for me. But we can change it in the future

Reply:
Just for unifying the name, we use INPROGRESS in other places for the retry cases.